src/Network/Wai/Middleware/Cors.hs

```diff
@@ -251,14 +251,15 @@ data CorsResourcePolicy = CorsResourcePolicy
 simpleCorsResourcePolicy ∷ CorsResourcePolicy
 simpleCorsResourcePolicy = CorsResourcePolicy
     { corsOrigins = Nothing
-    , corsMethods = simpleMethods
+    , corsMethods = simpleCorsMethods
     , corsRequestHeaders = []
     , corsExposedHeaders = Nothing
     , corsMaxAge = Nothing
     , corsVaryOrigin = False
     , corsRequireOrigin = False
     , corsIgnoreFailures = False
-    }
+    } where
+  simpleCorsMethods = ["HEAD", "GET", "PUT", "POST", "DELETE", "OPTIONS"]
```

Comment:
The [CORS standard](http://www.w3.org/TR/cors/) allows only `GET`, `HEAD`, and `POST` for *simple cross-origin requests*.

The policy that is provided by this function qualifies the resource for *simple cross origin requests*.

A server is free to define other CORS policies for the resources that it serves, but those don't necessarily qualify for *simple cross origin requests* as defined in the standard.